notification/event_type.go

```diff
@@ -0,0 +1,18 @@
+package notification
```

Comment:
Only some parts are exported. Can move these to `wot` package, under wot/discovery.go.